lib/datasource/local/settings_storage.dart

```diff
@@ -42,15 +44,19 @@ class _SettingsStorage {
 
   String get accountName => _preferences.getString(_kAccountName) ?? '';
 
+  List<String> get accountsList => _preferences.getStringList(_kAccountsList) ?? [];
+
   String? get privateKey => _privateKey;
 
+  List<String>? get privateKeysList => _privateKeysList;
+
   String? get passcode => _passcode;
 
   bool? get passcodeActive => _passcodeActive;
 
   bool? get biometricActive => _biometricActive;
 
-  bool get privateKeyBackedUp => _privateKeyBackedUp ?? false;
+  bool get privateKeyBackedUp => _privateKeyBackedUp ?? false; // <-- I don't see this being used anywhere in the app
```

Comment:
This flag privateKeyBackedUp It does not seem to be being used anywhere in any condition, nor internally from the storage .. only mapping operations are performed on it
![image](https://user-images.githubusercontent.com/42857405/132073631-ace66e74-92fb-4eee-ac89-67a89e10f5b9.png)

Reply:
This flag is used to warn users when their private key has not been backed up - should be used

Unless that function is broke?

Reply:
> Unless that function is broke?

Yes, it is, apparently we no longer have this feature in v2, it seems to me that it was removed in the cleanup, it was used to show a top bar in onboarding that reminded the user to create a backup of their account.

Should it be reinstated? 

I'm surprised it would be overlooked if it was an important feature.

https://github.com/JoinSEEDS/seeds_light_wallet/pull/182/files

Reply:
I dont think we implemented this feature. 

@JGDoh  Do we want it? If so, we need Issue and designs for it.

Reply:
> If so, we need Issue and designs for it.

Agree.